src/views/template/topbar.html

```diff
@@ -31,7 +30,7 @@
             <span id ="score">{{ config.score }}</span>

```

Comment:
Still, only the score displays. Not "Score"  itself.

Reply:
Do you have the text anywhere? I think this is just the number.

Also, `id="score"` (alignment)